packages/p/pahomqttcpp/xmake.lua

```diff
@@ -0,0 +1,35 @@
+package("pahomqttcpp")
+    set_homepage("https://github.com/eclipse/paho.mqtt.cpp")
+    set_description("Eclipse Paho MQTT C++ Client Library")
+    set_license("EPL v1.0")
+
+    add_urls("https://github.com/eclipse/paho.mqtt.cpp/archive/refs/tags/$(version).zip",
+             "https://github.com/eclipse/paho.mqtt.cpp.git")
+
+    add_versions("v1.2.0", "90c4d8ae4f56bb706120fddcc5937cd0a0360b6f39d5cd5574a5846c0f923473")
+    
+    add_deps("cmake")
+    add_deps("pahomqttc")
+
+    on_install("windows", "linux", "macos", function (package)
+        local pahomqttc = package:dep("pahomqttc")
+        
+        local configs = {"-DPAHO_WITH_SSL=FALSE"}
+        if package:is_plat("windows") then
+            table.insert(configs, "-DPAHO_MQTT_C_LIBRARIES=" .. pahomqttc:installdir("lib"))
```

Comment:
we should use pahomqttc:fetch(), because xmake maybe find it from system lib path.  installdir may be an empty directory.

https://github.com/xmake-io/xmake-repo/blob/ebe61bb7b8d7ecc11b3258747c80863f02084f74/packages/s/sol2/xmake.lua#L40